Baseball/Play Baseball.user.js

```diff
@@ -788,13 +829,33 @@ function Main() {
 			};
 		}));
 	});
+	const profitTag = CreateElement('div', divTag => {
+		divTag.id = 'Profit';
+		divTag.classList.add('col-sm-6');
+		divTag.append(CreateElement('h4', h3Tag => {
+			h3Tag.append('Income today after tips');
+		}));
+		divTag.append(CreateElement('p', pTag => {
+			pTag.id = 'ProfitValueAfterTips';
+			pTag.append("?");
+		}));
+		divTag.append(CreateElement('h4', h3Tag => {
+			h3Tag.append('Income today before tips');
+		}));
+		divTag.append(CreateElement('p', pTag => {
+			pTag.id = 'ProfitValueBeforeTips';
+			pTag.append("?");
+		}));
+	});
+
 	// Would only throw error on away page when there are no games currently available.
 	try {
 		const formTag = document.querySelector('input[name="token"]').parentElement;
 		if (!isHost) {
 			formTag.previousElementSibling.remove();
 		}
 		formTag.parentElement.insertBefore(divTag, formTag);
+		formTag.parentElement.insertBefore(profitTag, divTag.nextSibling);
```

Comment:
Also gotta duplicate this line below for, currently, 865 as it won't get added if the away player loads the page when there is no available games.